src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -129,7 +151,8 @@ class OptionsResolver implements Options
      *
      * @return $this
      *
-     * @throws AccessException If called from a lazy option or normalizer
+     * @throws AccessException      If called from a lazy option or normalizer
+     * @throws \ReflectionException
```

Comment:
should be removed (not for this PR at least, unrelated to what it does)
same below